restli-server/src/main/java/com/linkedin/restli/server/CursorCollectionResult.java

```diff
@@ -20,4 +20,16 @@ public CursorCollectionResult(final List<T> elements, CursorPagination paginatio
   {
     super(elements, null, pagination);
   }
+
+  /**
+   * Constructor
+   *
+   * @param elements       List of elements in the current page.
+   * @param total          Provides the total elements
+   * @param pagination     The cursor pagination metadata.
+   */
+  public CursorCollectionResult(final List<T> elements, final Integer total, CursorPagination pagination)
```

Comment:
Will this mix offset-based pagination with cursor-based pagination?

Reply:
The idea is to be able to return total number of entities if a partner wants it in cursor based pagination. 
Should we rather have a different parameter for it then?